pennylane/devices/qutrit_mixed/utils.py

```diff
@@ -68,6 +69,37 @@ def get_einsum_mapping(
     )
 
 
+def resquare_state(state, num_wires):
+    """
+    Given a non-flat, potentially batched state, flatten it to a square matrix.
+
+    Args:
+        state (TensorLike): A state that needs flattening
+        num_wires (int): The number of wires the state represents
+
+    Returns:
+        Tensorlike: A squared state, with an extra batch dimension if necessary
+    """
+    dim = QUDIT_DIM**num_wires
+    batch_size = math.get_batch_size(state, ((QUDIT_DIM,) * (num_wires * 2)), dim**2)
+    shape = (batch_size, dim, dim) if batch_size is not None else (dim, dim)
+    return math.reshape(state, shape)
+
+
+def get_num_wires(state, is_state_batched: bool = False):
+    """Finds the number of wires associated with a state
+
+    Args:
+        state (TensorLike): A state that needs flattening
```

Comment:
```suggestion
        state (TensorLike): A state that needs flattening
```
Update this docstring to reflect this method

Reply:
updated to better reflect method

Reply:
This docstring hasn't been updated; should reflect what the function is doing here